lib/bolt/config/transport/ssh.rb

```diff
@@ -13,12 +13,18 @@ class SSH < Base
         #       in schemas/bolt-transport-definitions.json
         OPTIONS = {
           "cleanup"            => { type: TrueClass,
+                                    external: true,
                                     desc: "Whether to clean up temporary files created on targets." },
           "connect-timeout"    => { type: Integer,
                                     desc: "How long to wait when establishing connections." },
+          "copy-command"       => { type: Array,
```

Comment:
This and `ssh-command` should have schema definitions and be added as properties to the SSH definition: https://github.com/puppetlabs/bolt/blob/master/schemas/bolt-transport-definitions.json